spec/features/admin_approve_delete_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature 'admin manages vendor page', %{
+  As an admin
+  I want to delete or approve a vendor
+  So that users can see appropriate vendors
+} do
+
+  let!(:category) { FactoryGirl.create(:category) }
+
+  scenario 'admin approves a valid vendor' do
+    member = FactoryGirl.create(:user, user_type: 'Admin')
+    sign_in(member)
+    visit vendors_path
+    fill_in 'Vendor Name', with: 'Crepe Truck'
+    select('Breakfast', from: 'vendor_category_ids')
+    click_button 'Add Vendor'
```

Comment:
Instead of filling out the form by hand, you should have a factory for this! Use that instead.